### ISSUE TITLE
Specifying configuration in-place changes global configuration

### ISSUE THREAD

**Issue by archSeer** [mazipan/vue-currency-filter#12]
```
{{balance | currency("", ",", 6, ".") }}
...
{{amount | currency}}
```

Now amount is formatted in the same way as balance, but it shouldn't be

**Comment by mazipan** [comment]
Yes, I see.
What do you expected condition @archSeer ?
Is reset to default configuration ?

**Comment by archSeer** [comment]
Yes, it should always use the default config unless I manually override it for that specific part of the page. Otherwise the display is order dependent (if I refactor the page and rearrange the elements it will suddenly render differently)

**Comment by mazipan** [comment]
Closed in v2.1.0
Check the demo page here : https://mazipan.github.io/vue-currency-filter/
